phi/tools/scrapegraph_tools.py

```diff
@@ -0,0 +1,68 @@
+import json
+from typing import Optional
+from phi.tools import Toolkit
+
+try:
+    from scrapegraph_py import Client
+except ImportError:
+    raise ImportError("`scrapegraph-py` not installed. Please install using `pip install scrapegraph-py`")
```

Comment:
I think it should be `pip install scrapegraphai`?

Reply:
Running the cookbook is giving me error. You are most probably not using the correct lib

Reply:
Hey, I am integrating the official sdk which is installed by `pip install scrapegraph-py` ;)
Please, try installing it and run again the cookbook. I have added an output image in the PR description.